tracer/src/Datadog.Trace.Tools.Shared/Windows/NativeMethods.cs

```diff
@@ -24,9 +24,9 @@
 using System.Runtime.InteropServices;
 using System.Text;
 
-namespace Datadog.Trace.Tools.dd_dotnet.Checks.Windows
+namespace Datadog.Trace.Tools.Shared.Windows
 {
-    internal static class NativeMethods
+    public static class NativeMethods
```

Comment:
You wouldn't need to change these if you used a shared library instead of a normal project 😉